examples/src/stories/slider/examples/range-slider-with-section-example.tsx

```diff
@@ -0,0 +1,138 @@
+import React from "react";
+
+import { RangeSlider } from "@axiscommunications/fluent-slider";
+import { makeStyles, tokens } from "@fluentui/react-components";
+
+const useCustomizedSliderStyles = makeStyles({
+  mark: {
+    height: "16px",
+    width: "1px",
+    backgroundColor: "white",
+  },
+  markLabel: {
+    color: tokens.colorPaletteRedForeground1,
+  },
+  track: {
+    backgroundColor: "gray",
+  },
+});
+
+export function RangeSliderWithSectionExample() {
+  const classes = useCustomizedSliderStyles();
+  return (
+    <RangeSlider
+      min={-40}
+      max={300}
+      defaultValue={[-10, 120]}
+      title="Click to change the value"
+      marks={[
+        {
+          value: 37,
+          // label:'BT', // "Body temperature"
```

Comment:
Do something with this. Remove or mirror to the string section below.

Reply:
Removed.